client/src/components/EmailFrontend.js

```diff
@@ -148,6 +150,8 @@ class EmailFrontend extends React.Component {
       completed_template,
     } = this.state;
 
+    const { top_border } = this.props;
```

Comment:
What's this needed for?

Reply:
![image](https://user-images.githubusercontent.com/25855114/91588382-0fb87f80-e926-11ea-8bd1-8fd1c5c303a4.png)

there's that border top on the report a problem thing. On the modal, there's a border along the title as well haha.
This is so that it doesn't show 2 borders unnecessarily